test-case-reporting/test/testing_utils.ts

```diff
@@ -25,3 +27,109 @@ export async function truncateAll(db: Database): Promise<void> {
   await db('jobs').truncate();
   await db('builds').truncate();
 }
+
+/**
+ * Fills the database with builds, test cases, jobs, and test runs.
+ * Stores 3 test cases.
+ * Stores 2 builds: the first one has 5 jobs and
+ * 15 test runs (one per (job, test case) pair).
+ * The second one has 1 job and 1 test run.
+ * @param db The database we want to fill.
+ */
+export async function fillDatabase(db: Database): Promise<void> {
+  let [buildId] = await db('builds')
+    .insert({
+      'commit_sha': 'deadbeefdeadbeef123123',
```

Comment:
Should I make this (and other fake commit SHAs) 40 characters long, like real commit SHAs?

Reply:
You can, shouldn't really matter though